src/auto-evo/EvolutionaryTree.cs

```diff
@@ -173,6 +187,9 @@ public void Clear()
         speciesNames.Clear();
         generationTimes.Clear();
         speciesNodes.Clear();
+        maxSpeciesId = 0;
+        latestGeneration = 0;
+        dragOffset = Vector2.Zero;
```

Comment:
If the user is currently dragging should this also reset the dragging status?

Reply:
I don't think so. I think dragging status should be preserved.

Reply:
But doesn't that then mostly negate the drag offset? Because if the user has dragged an entire screen, the tree is rebuilt, then the user moves the mouse again, this line basically has no effect as the drag offset will jump a ton at once.

Reply:
No. The drag offset won't jump a ton at once because every mouse move directly changes DragOffset a little - The later dragging will still be smooth.

Reply:
That's good then. I was thinking that this was the variable tracking where the cursor started the drag, but that's obviously a different variable.